Remotion/ObjectBinding/Web/UI/Controls/Validation/BocListListValidationFailureHandler.cs

```diff
@@ -0,0 +1,53 @@
+﻿// This file is part of the re-motion Core Framework (www.re-motion.org)
+// Copyright (c) rubicon IT GmbH, www.rubicon.eu
+//
+// The re-motion Core Framework is free software; you can redistribute it
+// and/or modify it under the terms of the GNU Lesser General Public License
+// as published by the Free Software Foundation; either version 2.1 of the
+// License, or (at your option) any later version.
+//
+// re-motion is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with re-motion; if not, see http://www.gnu.org/licenses.
+// using NUnit.Framework;
+//
+using Remotion.ObjectBinding.Web.UI.Controls.BocListImplementation;
+using Remotion.ServiceLocation;
+using Remotion.Utilities;
+
+namespace Remotion.ObjectBinding.Web.UI.Controls.Validation
+{
+  /// <summary>
+  /// Handles all validation failures for the <see cref="IBusinessObjectProperty"/> of the <see cref="IBocList"/>. Should always be the first handler to run.
+  /// </summary>
+  /// <seealso cref="IBocListValidationFailureHandler"/>
+  [ImplementationFor(typeof(IBocListValidationFailureHandler), Lifetime = LifetimeKind.Singleton, RegistrationType = RegistrationType.Multiple, Position = c_position)]
+  public class BocListListValidationFailureHandler : IBocListValidationFailureHandler
+  {
+    private const int c_position = -100;
```

Comment:
```suggestion
  [ImplementationFor(typeof(IBocListValidationFailureHandler), Lifetime = LifetimeKind.Singleton, RegistrationType = RegistrationType.Multiple, Position = Position)]
  public class BocListListValidationFailureHandler : IBocListValidationFailureHandler
  {
    public const int Position = -100;
```

Reply:
Why do we need them public?

Reply:
So other users can define their position relative to ours.

Reply:
OtherImpl{ const int Position = BaseImpl.Position + 10; }